src/execution/ValueSetHelper.ts

```diff
@@ -107,33 +107,51 @@ export function getMissingDependentValuesets(
     throw new UnexpectedResource('Expected measure bundle to contain entries');
   }
   const libraryEntries = measureBundle.entry?.filter(
-    e => e.resource?.resourceType === 'Library' && e.resource.dataRequirement
+    e => e.resource?.resourceType === 'Library' && (e.resource.dataRequirement || e.resource.relatedArtifact)
```

Comment:
Should we worry about any relatedArtifact(s) on the Measure resource as well?

Reply:
It actually does not look like ValueSets are referenced in the Measure resource by itself. So just looking at the Library is appropriate.